api/src/main/java/io/strimzi/api/kafka/model/Zookeeper.java

```diff
@@ -23,12 +23,14 @@
 )
 @JsonPropertyOrder({ "replicas", "image", "storage",
         "livenessProbe", "readinessProbe", "jvmOptions",
-        "affinity", "metrics"})
+        "affinity", "metrics", "stunnelImage"})
```

Comment:
The one we use for the rack feature is `brokerRackInitImage` (i.e. the fact it's used for an init container is apparent from the name). In this case we have a sidecar. If we think the nature of the container is important when specifying the image name we should name the property accordingly. In not, perhaps we should rename `brokerRackInitImage`.

Reply:
Maybe we don't need to rename if we consider that `XXXXImage` is something used as image for a "normal" container while `XXXXInitImage` is related to an "init" container.

Reply:
As I suggested elsewhere ... I would probably name it `tlsSidecarImage`.